### PR TITLE
DOC-1822: Edit mailing/billing address UX changes

### PR DESCRIPTION
**Pull request by cmilesb**
Staged links: 
https://docs.redis.com/staging/jira-doc-1822/rc/accounts/account-settings/
https://docs.redis.com/staging/jira-doc-1822/rc/billing-and-payments/

### CODE REVIEW

File: content/rc/billing-and-payments.md [hunk 1]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+    {{< /note >}}
+
+4.  Select the **Add Credit Card** button to save your changes.
+
+    {{<image filename="images/rc/button-billing-save-card.png" width="150px" alt="Use the Save Card button to save new payment details." >}}{{< /image >}}
+
+## Edit billing address
 
-2.  Use the admin console menu to select **Billing & Payments** and then select the **Payment Methods** tab.
+To edit the account billing address:
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:109]:
```suggestion
To edit the Redis Cloud account billing address:
```

File: content/rc/billing-and-payments.md [hunk 2]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
 
-3.  Select **Add credit card** and then provide the details.
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:111]: Another options for these types of steps 

```suggestion
1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** > **Payment Methods**.
```

File: content/rc/billing-and-payments.md [hunk 3]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
 
-    {{<image filename="images/rc/billing-add-credit-card.png" width="300px" alt="Select Add Credit Card to add a new payment method" >}}{{< /image >}}
+2. Select the **Credit Card options** menu and select **Edit billing address**.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:113]: Same type of suggestion to cut length as above: 
```suggestion
2. Select **Credit Card options** > **Edit billing address**.
```

File: content/rc/billing-and-payments.md [hunk 4]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
 
-    {{<image filename="images/rc/billing-update-billing-address.png" width="300px" alt="Deactive the Use account address slider to specfy a different billing address." >}}{{< /image >}}
+3. Enter the new billing address. You must re-enter your payment method details to confirm your address change.
 
-5.  Use the **Save Card** button to save your changes.
+    {{<image filename="images/rc/account-settings-change-billing-address.png" alt="The Edit account billing address screen." width="80%">}}{{< /image >}}
 
-    {{<image filename="images/rc/button-billing-save-card.png" width="100px" alt="Use the Save Card button to save new payment details." >}}{{< /image >}}
+    {{< note >}}
+Changing the billing address to your account will remove any payment methods associated with the old billing address. See [Add payment method](#add-payment-method) to learn how to add a payment method back to your account.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:122]:
```suggestion
Changing the billing address for your account will remove any payment methods associated with the old billing address. See [Add payment method](#add-payment-method) to learn how to add a payment method back to your account.
```

File: content/rc/billing-and-payments.md [hunk 5]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+    {{< /note >}}
+
+## Edit mailing address
+
+To edit the mailing address associated with a payment method:
+
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:129]:
```suggestion
1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** > **Payment Methods** .
```

File: content/rc/billing-and-payments.md [hunk 6]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+
+2. Select the **Credit Card options** menu and select **Edit mailing address**.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:131]:
```suggestion
2. Select  **Credit Card options** > **Edit mailing address**.
```

File: content/rc/billing-and-payments.md [hunk 7]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+    {{<image filename="images/rc/billing-change-mailing-address.png" alt="The Edit mailing address screen." width="80%">}}{{< /image >}}
 
 ## Apply coupon
 
 Coupons apply credits to your Redis Enterprise Cloud account.  To redeem a coupon:
 
-1. Sign in to the Redis Cloud [admin portal](https://app.redislabs.com/).  (Create an account if you don't already have one.)
-
-2.  Use the admin console menu to select **Billing & Payments** and then select the **Credits** tab.
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then select the **Credits** tab.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:143]:
```suggestion
1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** > **Credits**.
```

File: content/rc/billing-and-payments.md [hunk 8]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
 
-3.  Enter the coupon code and then select the **Apply** button.
+2.  Enter the coupon code and then select the **Apply** button.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:145]:
```suggestion
2.  Enter the coupon code and select the **Apply** button.
```

File: content/rc/billing-and-payments.md [hunk 9]
```diff
@@ -72,13 +72,11 @@ The following tabs are available:
 
 To download an invoice:
 
-1. Sign in to the Redis Cloud [admin portal](https://app.redislabs.com/).  (Create an account if you don't already have one.)
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then make sure the Billing History tab is selected.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:75]:
```suggestion
1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and make sure the Billing History tab is selected.
```

File: content/rc/billing-and-payments.md [hunk 10]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
 
 To add a new payment method:
 
-1. Sign in to the Redis Cloud [admin portal](https://app.redislabs.com/).  (Create an account if you don't already have one.)
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:89]:
```suggestion
1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** > **Payment Methods** .
```

File: content/rc/billing-and-payments.md [hunk 11]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+
+2.  Select **Add credit card** and then provide the details.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:91]:
```suggestion
2.  Select **Add credit card** and add the credit card details.
```

File: content/rc/billing-and-payments.md [hunk 12]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
-2.  Use the admin console menu to select **Billing & Payments** and then select the **Credits** tab.
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then select the **Credits** tab.
 
-3.  Enter the coupon code and then select the **Apply** button.
+2.  Enter the coupon code and then select the **Apply** button.
 
     {{<image filename="images/rc/button-billing-payments-apply.png" width="80px" alt="Use the Apply button to redeem a coupon." >}}{{< /image >}}
 
-The value of the coupon is applied to your account when accepted.  
+    The value of the coupon is applied to your account when accepted.
```
Comment (kaitlynmichael) [inline content/rc/billing-and-payments.md:149]:
```suggestion
    The value of the coupon applies to your account when accepted.  
```

Comment (kaitlynmichael) [reply content/rc/billing-and-payments.md:149]: Active voice change

**Review by kaitlynmichael — APPROVED** [review]
Great job! Very clear. I added some suggestions to shorten the language or change to active voice.

File: content/rc/billing-and-payments.md (hunk 9 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:75]:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments > Billing History**.
```

Or you could remove the "Billing History" part and just specify this tab in the next step:

```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments**.
```

File: content/rc/billing-and-payments.md [hunk 13]
```diff
@@ -72,13 +72,11 @@ The following tabs are available:
 
 To download an invoice:
 
-1. Sign in to the Redis Cloud [admin portal](https://app.redislabs.com/).  (Create an account if you don't already have one.)
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then make sure the Billing History tab is selected.
 
-2.  Use the admin console menu to select **Billing & Payments** and then make sure the Billing History tab is selected.
+2.  Locate and select the invoice.
```
Comment (rrelledge) [inline content/rc/billing-and-payments.md:77]: If you choose to remove "Billing History" from the previous step:
```suggestion
2.  From the **Billing History** tab, locate and select the invoice.
```

File: content/rc/billing-and-payments.md (hunk 10 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:89]:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```

File: content/rc/billing-and-payments.md (hunk 2 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:111]:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```

File: content/rc/billing-and-payments.md (hunk 5 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:129]:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments** and then select the **Payment Methods** tab.
```

File: content/rc/billing-and-payments.md (hunk 7 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:143]:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments** and then select the **Credits** tab.
```

File: content/rc/accounts/account-settings.md [hunk 14]
```diff
@@ -43,13 +43,21 @@ You cannot change the email address associated with a Redis Cloud account.  Inst
...
+The **Account address** section shows the billing address associated with the current Redis Cloud account and the current time zone. 
 
 To update the time zone, select the desired time zone from the **Time zone** drop-down.
 
 In addition, this section may include fields unique to your location.  For example, certain regions require tax IDs or other regulatory details.
 
-When updating details in this section, pay particular attention to prompts and error messages.
+Select **Edit** to change the account's billing address. You must re-enter your payment method details to confirm your address change. 
+
+   {{<image filename="images/rc/account-settings-edit-address-button.png" alt="Select the Edit button to change the account's billing addresss." width="75%">}}{{< /image >}}
```
Comment (rrelledge) [inline content/rc/accounts/account-settings.md:54]: This image is broken. Looks like there is a typo in the actual filename: an extra "d" in "address".
```suggestion
   {{<image filename="images/rc/account-settings-edit-address-button.png" alt="Select the Edit button to change the account's billing address." width="75%">}}{{< /image >}}
```

File: content/rc/billing-and-payments.md [hunk 15]
```diff
@@ -88,34 +86,66 @@ The invoice is downloaded as an Acrobat PDF file.  Use your browser's download f
...
+
+3.  If you need to change the account billing address for this card, activate the **Change Address** slider and enter the new billing address.
+
+    {{<image filename="images/rc/billing-update-billing-address.png" width="80%" alt="Deactivate the Use account address slider to specify a different billing address." >}}{{< /image >}}
+
+    {{< note >}}
+Changing the billing address to your account will remove any payment methods associated with the old billing address. 
+    {{< /note >}}
+
+4.  Select the **Add Credit Card** button to save your changes.
```
Comment (rrelledge) [inline content/rc/billing-and-payments.md:103]:
```suggestion
4.  Select the **Add credit card** button to save your changes.
```

File: content/rc/billing-and-payments.md (hunk 3 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:113]:
```suggestion
2. Select the **Credit card options** menu and select **Edit billing address**.
```

File: content/rc/billing-and-payments.md (hunk 6 above)
Comment (rrelledge) [inline content/rc/billing-and-payments.md:131]:
```suggestion
2. Select the **Credit card options** menu and select **Edit mailing address**.
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Oh, I also noticed a broken image that needs to be fixed.

**Review by rrelledge — APPROVED** [review]
Looks good to me!

**Pull request merged**